frontend/src/entry-points/shop.tsx

```diff
@@ -218,41 +225,43 @@ const Shop: Component = () => {
   }
 
   return (
-    <div class="c-grid p-3">
-      <div class="min-width">
-        <div class="flex justify-between gap-2">
-          <h1 class="text-2xl font-bold">Webxdc Appstore</h1>
-          <div class="rounded-xl bg-gray-100 p-2 unimportant text-gray-500">
-            <Show when={isUpdating()} fallback={
-              <button class="flex items-center gap-2" onclick={handleUpdate}>
-                <span>{formatDuration(timeSinceLastUpdate(), { delimiter: ',' }).split(',')[0]} ago</span>
-                <div class="border border-blue-500 rounded" i-material-symbols-sync></div>
-              </button>
-            }>
-              <div class="flex items-center gap-2">
-                <span>Updating..</span>
-                <div class="loading-spinner border border-blue-500 rounded" i-material-symbols-sync></div>
-              </div>
-            </Show>
+    <OutdatedView critical={updateNeeded()}>
```

Comment:
This needs to be done for every xdc the bot serves. Atm it's only implemented for the shop wich is needed for MVP.